### ISSUE TITLE
any links in any services inside Franz won't open since I installed the 5 beta.

### ISSUE THREAD

**Issue by krysalia** [meetfranz/franz#471]
any links in any services inside Franz won't open since I installed the 5 beta.

### Expected Behavior

When clicked, a link inside one of Franz services should open the default browser with the page of that link. 

It worked fine till I installed the version 5. Then in the 5 I noticed I could not open any links. 
I reverted to the 3.xx and then updated it automatically to the 4.0.4, hopefully to have the links working again. they are not :/. the click simply doesn't respond as if it was plain text. 

I'm currently on 4.0.4 and I do not mind switching to the 5, I only went back because of the links problem. I just really need the links to open. 

### Context

I closed franz and installed the 5 version. then it opened and no links worked. i reverted to the old version I had on hand and then updated it, in order to go back when the links worked. it failed. they do not work no matter the version now. 

### Your Environment

windows seven x64 

* Franz Version used:
* Operating System and version: 4.04, 5, 3.xx, 4.04

**Comment by adlk** [comment]
Do you use chrome? If yes, did it open a new tab in chrome that was just blank?

**Comment by krysalia** [comment]
Hello, I use palemoon as a default browser, but I tried to switch the default browser to vivaldi which is chormium based and Franz did nothing more. It acts with links as if it was plain text, but only the link that open a browser. For example in tweetdeck, the link "open that thread" works and opens the thread inside tweetdeck. this is the same for opening the tweets inside tweetdeck. but if I click on the permalink, it won't do a thing.

**Comment by LeonFedotov** [comment]
@adlk yes i have this issue

**Comment by lflrocha** [comment]
I have the same issue. Franz 4.0.4 was working fine. After update to 5.0.0-beta.14.451 nothing happen when clicking on links. I use Firefox as my default browser.

**Comment by Zearderes** [comment]
Same for me with Firefox as default browser.
Franz's version : 5.0.0-beta.14

**Comment by krysalia** [comment]
At least I'm happy to see that I'm not alone with this --'...

**Comment by kfirjz** [comment]
I am using chrome under mac and it opens a new chrome window every time I click on a link with no url to follow

**Comment by krysalia** [comment]
I tried a few things, such as updating various services in windows, and/or java etc, and the links still do not work in the 5 version. 
about the picture links, it's even more odd : when I click on a picture in tweetdeck theatre mode it opens.... A MiniFranz window, with the tweet page opened inside. #WTF... it acts just like a browser when it's not needed, and NOT like a browser when it should (to open links). 
the program is now totally unusable for me :( . It's a pity, I used to spend my days on this and I would gladly have bought the premium for the beta. 

Actually that's pretty ironic because the links problems happened first when I tried to click where it was about giving money and support/premium features, if I remember correctly :/. 

(screenshot uploaded : the MiniFranz Unusable Feature.)
![screenshot_721](https://user-images.githubusercontent.com/34556315/34445530-8b8d514a-ecd4-11e7-890e-36d3333168b6.png)
